app/controllers/idv_controller.rb

```diff
@@ -32,6 +33,22 @@ def activated
 
   private
 
+  def clear_session
+    user_session['idv/doc_auth'] = {}
+    user_session['idv/in_person'] = {}
+    user_session['idv/inherited_proofing'] = {}
+    idv_session.clear
+    Pii::Cacher.new(current_user, user_session).delete
+  end
```

Comment:
Is there a reason why we aren't directing to the `#destroy` action on `Idv::SessionsController`, where this session clearing is already implemented?

Reply:
the destroy action is doing more than we need for this situation. I'll pull the `clear_session` method into a separate helper file.